notebooks/README.md

```diff
@@ -53,6 +53,7 @@ See [Notebooks from NVIDIA's RAPIDS](../cloud-devops-infra/gpus/rapids.md#notebo
     - [Spacy/Texacy Examples](../examples/better-nlp/notebooks/google-colab/better_nlp_spacy_texacy_examples.ipynb)
     - [Summarisers Examples](../examples/better-nlp/notebooks/google-colab/better_nlp_summarisers.ipynb)
 - [Bag of words notebook](../blogs/keras-bag-of-words-expanded-version/data-scripts-notebooks/)
+- [TFIDF notebook](../I shall update here.)
```

Comment:
@NavyaTatikonda Yes pls proceed - sensible place, do you want to close the other PR?

Reply:
Yes, doing it now :)

Reply:
Will you merge this PR?

Reply:
Let me know when it\s ready with the link

Reply:
@NavyaTatikonda Any update on this?

Reply:
I have the doc ready in Scikit-learn, but not in keras.
Can I still proceed and update it?

Reply:
ok

Reply:
Submit whatever is ready